### PR TITLE
NMO:  Release branch lanes + OCP 4.13 for main

### PR DESCRIPTION
**Pull request by razo7**
- Simplify e2e test name (remove OCP version suffix in E2E test name)
- Add OCP 4.13 main lane
- Create release-0.14 lanes
- Run `make update`

### CODE REVIEW

File: ci-operator/config/medik8s/node-maintenance-operator/medik8s-node-maintenance-operator-main__4.13.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      product: ocp
+      stream: ci
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 500m
+      memory: 1000Mi
+tests:
+- as: openshift-e2e-ocp-4-13
```
Comment (slintes) [inline ci-operator/config/medik8s/node-maintenance-operator/medik8s-node-maintenance-operator-main__4.13.yaml:34]: do you want to remove `-ocp-4-13` here as well?

**Comment by slintes** [comment]
/lgtm
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-main-4.11-openshift-e2e
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-main-4.13-openshift-e2e

**Comment by slintes** [comment]
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-release-0.14-4.12-openshift-e2e

**Comment by razo7** [comment]
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-release-main-4.13-openshift-e2e

**Comment by razo7** [comment]
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-main-4.13-openshift-e2e

**Comment by razo7** [comment]
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-main-4.13-openshift-e2e

**Comment by razo7** [comment]
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-main-4.13-openshift-e2e

**Comment by razo7** [comment]
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-main-4.13-openshift-e2e

**Comment by razo7** [comment]
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-main-4.13-openshift-e2e

**Comment by razo7** [comment]
/pj-rehearse pull-ci-medik8s-node-maintenance-operator-main-4.13-openshift-e2e

**Pull request merged**